exercises/beer-song/description.md

```diff
@@ -1,4 +1,4 @@
-Produce the lyrics to that beloved classic, that field-trip favorite: 99 Bottles of Beer on the Wall.
```

Comment:
`lyrics` was OK.

Reply:
> #983 calls for standardizing the descriptions as well as the property name 

Hmm I didn't notice that :)
I like lyrics better than verses here tbh.